src/backend/renderer/mod.rs

```diff
@@ -216,6 +218,12 @@ pub trait Frame {
     /// Output transformation that is applied to this frame
     fn transformation(&self) -> Transform;
 
+    /// Wait for the [`SyncPoint`] to be signaled
+    fn wait(&mut self, sync: &sync::SyncPoint) -> Result<(), Self::Error> {
```

Comment:
Why is this an api of `Frame` and not `Renderer` (or both?)

Reply:
The intend behind this was that it only makes sense if you are currently rendering something to insert a server wait. I can add the same function to `Renderer` if you want?

Reply:
added `wait` to the `Renderer` trait